tools/clang/lib/Frontend/FrontendActions.cpp

```diff
@@ -478,6 +478,7 @@ namespace {
 #define VALUE_LANGOPT(Name, Bits, Default, Description) \
       Out.indent(4) << Description << ": " << LangOpts.Name << "\n";
 #define BENIGN_LANGOPT(Name, Bits, Default, Description)
+#define BENIGN_LANGOPT_BOOL(Name, Default, Description)
```

Comment:
Here as well. I don't know how we get out of defining LANGOPT_BOOL

Reply:
Added default define for LANGOPT_BOOL.